crates/libcgroups/src/v2/devices/bpf.rs

```diff
@@ -24,26 +24,33 @@ pub mod prog {
     // TODO: consider use of #[mockall_double]
     #[cfg(test)]
     use crate::v2::devices::mocks::mock_libbpf_sys::{
-        bpf_load_program, bpf_prog_attach, bpf_prog_detach2, bpf_prog_get_fd_by_id, bpf_prog_query,
+        bpf_prog_attach, bpf_prog_detach2, bpf_prog_get_fd_by_id, bpf_prog_load,
+        bpf_prog_load_opts, bpf_prog_query,
     };
     #[cfg(not(test))]
     use libbpf_sys::{
-        bpf_load_program, bpf_prog_attach, bpf_prog_detach2, bpf_prog_get_fd_by_id, bpf_prog_query,
+        bpf_prog_attach, bpf_prog_detach2, bpf_prog_get_fd_by_id, bpf_prog_load,
+        bpf_prog_load_opts, bpf_prog_query,
     };
 
     pub fn load(license: &str, insns: &[u8]) -> Result<RawFd> {
         let insns_cnt = insns.len() / std::mem::size_of::<bpf_insn>();
         let insns = insns as *const _ as *const bpf_insn;
+        let opts = libbpf_sys::bpf_prog_load_opts {
```

Comment:
I'm pretty sure we can use just the default without modifying anything, as all fields are c_uints, whose defaults will be zero nonetheless. Have added this here for initial, can remove later